air/src/aux_table/bitwise_pow2/bitwise/mod.rs

```diff
@@ -279,6 +270,11 @@ trait EvaluationFrameExt<E: FieldElement> {
     fn bit_decomp(&self) -> &[E];
     /// Gets the entire range of decomposed input values for `a` and `b` in the next row.
     fn bit_decomp_next(&self) -> &[E];
+    /// Gets the value of the aggregated output in the previous row.
+    fn output_prev(&self) -> E;
+    /// Gets the value of the aggregated output of the current row, or
+    /// the previous row with respect to the next row.
+    fn output_prev_next(&self) -> E;
```

Comment:
Since we're no longer using `bit_decomp_next` (right above this), it can be removed. The same is true for `output_next` (below).